.travis.yml

```diff
@@ -5,9 +5,13 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
```

Comment:
Another thing - the dbname is apparently supposed to be the last option of the command. This isn't a problem on osx but on windows it had to be last.